data-node/src/main/java/org/graylog/datanode/configuration/variants/OpensearchSecurityConfiguration.java

```diff
@@ -159,7 +193,7 @@ protected ImmutableMap<String, String> commonSecureConfig() {
         config.put("plugins.security.allow_default_init_securityindex", "true");
         //config.put("plugins.security.authcz.admin_dn", "CN=kirk,OU=client,O=client,L=test,C=de");
 
-        config.put("plugins.security.audit.type", "internal_opensearch");
+        //config.put("plugins.security.audit.type", "internal_opensearch");
```

Comment:
This comment can be removed, I think.

Reply:
I put it back in as the default, changed it for debugging purposes and removed those settings. see above